frontend/components/events/EventCard.tsx

```diff
@@ -38,42 +37,46 @@ export default function EventCard(props: EventCardProps) {
   } = props;
 
   const cardContent = (
-    <div className="bg-white rounded-lg text-left border-gray-300 border w-full md:w-[300px] xl:w-[290px] 2xl:w-[320px] hover:cursor-pointer overflow-hidden">
+    <div className="bg-white text-left w-full hover:cursor-pointer">
       {loading ? (
         <div>
           <LoadingSkeletonEventCard />
         </div>
       ) : (
         <>
           <div
-            className="h-36 w-full"
+            className="w-full"
             style={{
               backgroundImage: `url(${image})`,
               backgroundSize: "cover",
               backgroundPosition: "center center",
+              aspectRatio: "1/1",
```

Comment:
I like the use of aspect ratio because it makes everything uniform. However, I'm thinking we want aspect ratios a bit more like video or photo formats, something like 4/3 or 16/9 🤔 ? Currently the event cards on the main page look a bit tall.